package.json

```diff
@@ -23,6 +23,8 @@
     "fast-text-encoding": "^1.0.0",
     "gaxios": "^4.0.0",
     "gcp-metadata": "^4.2.0",
+    "google-auth-library": "^7.0.4",
```

Comment:
If these are just being used in samples, you would need to add them to `samples/package.json`.  There should already be a reference to `google-auth-library` there, since it gets linked in as part of the tests.